estuary/api/v1.py

```diff
@@ -218,3 +222,42 @@ def _get_partial_stories(query, reverse=False):
         all_results.append(rv)
 
     return jsonify(all_results)
+
+
+@api_v1.route('/siblings/<resource>/<uid>')
+def get_siblings(resource, uid):
+    """
+    Get siblings of next/previous node that are correlated to the node in question.
+
+    :param str resource: a resource name that maps to a neomodel class
+    :param str uid: the value of the UniqueIdProperty to query with
+    :return: a Flask JSON response
+    :rtype: flask.Response
+    :raises NotFound: if the item is not found
+    :raises ValidationError: if an invalid resource was requested
+    """
+    # If last is True, we fetch siblings of the next node, previous node otherwise
+    last = request.args.get('last', False)
+
+    item = get_neo4j_node(resource, uid)
```

Comment:
I wonder if this is actually necessary query since you already know the label from `resource` and if no results are returned below, you know the item doesn't exist.

Reply:
Actually, you only use the label from the `item` so I think it could actually be refactored

Reply:
I do need the `item` because I am sending it to `get_correlated_nodes` and it uses the Neo4j id of that node. So I think we will have to keep this

Reply:
This is fine, it's just that in `get_correlated_nodes` you only use the label.

Reply:
In lines #251 and #253, I send `item` as `next_node` to (now)`get_correlated_nodes`. And while querying Neo4j, we use the `id` in #238 of general.py